funnel/views/account.py

```diff
@@ -366,7 +366,7 @@ def account_edit(self, newprofile: bool = False) -> ReturnView:
                 submit=_("Continue"),
                 message=Markup(
                     _(
-                        "Hello, <strong>{fullname}</strong>. Please spare a minute to fill"
+                        "Hello, {fullname}. Please spare a minute to fill"
                         " out your account"
```

Comment:
Also re-wrap the lines. Our line length is 88 characters. Fit as much as will fit per line.